satpy/dataset/metadata.py

```diff
@@ -49,6 +50,10 @@ def combine_metadata(*metadata_objects, average_times=True):
         return info_dicts[0].copy()
 
     shared_keys = _shared_keys(info_dicts)
+    try:
+        shared_keys.remove('raw_metadata')
```

Comment:
I think you can use `.discard` and remove the try/except.

Reply:
Thanks for the hint!